server/src/main/java/org/elasticsearch/cluster/ClusterState.java

```diff
@@ -638,11 +634,11 @@ public Iterator<? extends ToXContent> toXContentChunked(ToXContent.Params outerP
             chunkedSection(
                 metrics.contains(Metric.NODES),
                 (builder, params) -> builder.startArray("transport_versions"),
```

Comment:
We can't remove the `transport_versions` section (that would be a breaking change) but it probably makes sense to add a new `nodes_versions` section which contains the same info but which can grow as we add more versions to the wrapper.

Reply:
Does it make sense to keep using `Metric.NODES` for this kind of version information? I see that we have `Metric.VERSION` but that looks like it's for the version of the cluster state, rather than versions per node.

Reply:
Yes I think `NODES` is fine.